hol/05_adding_rag.md

```diff
@@ -97,10 +97,15 @@ class EmbeddingStoreFactory implements Supplier<EmbeddingStore<TextSegment>> {
 
 To connect our embedding store to the **AI Service**, we need a **Content Retriever**. Instead of writing Java code, Helidon allows configuring it directly via YAML.
 
-**Modify `application.yaml` and add the following:**
+**Modify `application.yaml` and add the RAG configuration part:**
 
 ```yaml
 langchain4j:
+  open-ai:
```

Comment:
open-ai configuration should be in the config already at this stage. The snippet contains only the bits to add.